client/src/components/productDetails/SizeSelector.jsx

```diff
@@ -1,33 +1,72 @@
 import React from 'react';
 import QuantitySelector from './QuantitySelector.jsx';
+import AddToCart from './AddToCart.jsx';
 
 const { useEffect, useState } = React;
 
 function SizeSelector({ selectedStyle }) {
   const [skus, setSkus] = useState({});
   const [selectedSizeAmount, setSelectedSizeAmount] = useState(0);
+  const initialSelection = { size: '', quantity: '' };
+  const [selection, setSelection] = useState(initialSelection);
 
   useEffect(() => {
     if (selectedStyle) {
-      console.log('selectedStyle', selectedStyle);
       setSelectedSizeAmount(0);
       setSkus(selectedStyle.skus);
+      setSelection(initialSelection);
```

Comment:
do you need to setSelection and setSelectedSizeAmount in the useEffect as I see on line 9 and 11 you already assigned the same values as initial states?